HelpfulItem.h

```diff
@@ -10,6 +10,5 @@
 struct HelpfulItem : Item
```

Comment:
Don't include what you don't use.
check the header files included here.
- [ ] revise